File: mpp/Lab10/javaPb9/networking/src/main/java/ro/mpp2024/DTO/PunctajParticipantDTO.java
package ro.mpp2024.DTO;

import java.io.Serializable;

public class PunctajParticipantDTO implements Serializable {
    private Long idParticipant;
    private String nume,prenume;
    private int varsta;
    private long punctaj;

    public PunctajParticipantDTO() {
    }

    public PunctajParticipantDTO(long idParticipant, String nume, String prenume, int varsta, long punctaj) {
        this.idParticipant = idParticipant;
        this.nume = nume;
        this.prenume = prenume;
        this.varsta = varsta;
        this.punctaj = punctaj;
    }

    public Long getIdParticipant() {
        return idParticipant;
    }

    public String getNume() {
        return nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public int getVarsta() {
        return varsta;
    }

    public long getPunctaj() {
        return punctaj;
    }

    @Override
    public String toString() {
        return "PunctajParticipantDTO{" +
                "idParticipant=" + idParticipant +
                ", nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                ", varsta=" + varsta +
                ", punctaj=" + punctaj +
                '}';
    }
}

File: mpp/Lab11/csharpPb9/networking/DTO/PunctajParticipantDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace networking.DTO
{
    [Serializable]
    internal class PunctajParticipantDTO 
    {
        [JsonPropertyName("idParticipant")]
        public long idParticipant { get; set; }

        [JsonPropertyName("nume")]
        public string nume { get; set; }

        [JsonPropertyName("prenume")]
        public string prenume { get; set; }

        [JsonPropertyName("varsta")]
        public int varsta { get; set; }

        [JsonPropertyName("punctaj")]
        public long punctaj { get; set; }

        public PunctajParticipantDTO() { }

        public PunctajParticipantDTO(long idParticipant, String nume, String prenume, int varsta, long punctaj)
        {
            this.idParticipant = idParticipant;
            this.nume = nume;
            this.prenume = prenume;
            this.varsta = varsta;
            this.punctaj = punctaj;
        }

        public override string ToString()
        {
            return $"ParticipantDTO [idParticipant={idParticipant}, nume={nume}, prenume={prenume}, varsta={varsta}, punctaj={punctaj}]";
        }
    }
}
